frontend/packages/ceph-storage-plugin/src/constants/ocs-install.ts

```diff
@@ -44,3 +44,7 @@ export const infraProvisionerMap = {
   aws: 'kubernetes.io/aws-ebs',
   vsphere: 'kubernetes.io/vsphere-volume',
 };
+
+export const NO_SCHEDULE_EFFECT = 'NoSchedule';
+export const PREFER_NO_SCHEDULE_EFFECT = 'PreferNoSchedule';
+export const NO_EXECUTE_EFFECT = 'NoExecute';
```

Comment:
These are not specific to the ceph plugin and should be defined elsewhere. Probably `selectors/node.ts`.

Reply:
@spadgett Agreed. Creating a `node.ts` file at `/packages/console-shared/src/constants` for all node constants.